USAGE

```diff
@@ -35,6 +35,10 @@ Which is (NBR_PARCEL) * (NBR_PARCEL-1) / 2. Running connectoflow with a freesurf
 parcellation is A LOT faster than with Brainnetome, which is A LOT faster than
 with Glasser, make sure to test & optimize calls before launching on clusters.
 
+If providing a lesion mask, the nifti should be a uint8 binary image representing
+the voxel that are considered lesions. These lesions should have been taken into account
+when generating the tractogram (to allow tractography within the lesions).
```

Comment:
Please explicitly mention that lesion_mask must be in the same space as the diffusion data (dwi, fodf, tractogram, ...).